lib/rules/media-query-no-invalid/README.md

```diff
@@ -62,3 +62,39 @@ The following patterns are _not_ considered problems:
 ```css
 @media (color) and ((hover) or (width)) {}
 ```
+
+## Optional secondary options
+
+### `ignoreFunctions: ["/regex/", /regex/, "string"]`
+
+Ignore the specified functions.
+
+For example, with `true`.
```

Comment:
This is not necessary for `true`/`null` rules.

Reply:
Right. `true` is obvious. We can remove this line and simplify the description. 👍🏼

Reply:
I've fixed it.
[Update README.md](https://github.com/stylelint/stylelint/pull/8060/commits/fdf7de1a74e8b46739ac864b0ce46c2098262cfd)